mana_decay_provider_test.go

```diff
@@ -297,68 +297,68 @@ func TestManaDecay_PotentialMana(t *testing.T) {
 
 func TestManaDecay_Rewards(t *testing.T) {
 	type test struct {
-		name              string
-		rewards           iotago.Mana
-		epochIndexReward  iotago.EpochIndex
-		epochIndexClaimed iotago.EpochIndex
-		result            iotago.Mana
-		wantErr           error
+		name         string
+		rewards      iotago.Mana
+		epochReward  iotago.EpochIndex
```

Comment:
rename `epochReward` and `epochClaimed` to `rewardEpoch`and `claimedEpoch` to be in line with the other names where we always postfix with 'Epoch' or 'Slot'.